vendor/k8s.io/client-go/informers/storage/v1/csidriver.go

```diff
@@ -22,7 +22,7 @@ import (
 	"context"
 	time "time"
 
-	storagev1 "k8s.io/api/storage/v1"
+	storagev1beta1 "k8s.io/api/storage/v1"
```

Comment:
This is a weird vendor. Why is it called storagev1beta1 when the api is v1?
Also, what caused this to vendor in the first place?

Reply:
Oops, this might have a mistake I made when changing back to storagev1beta1. Let me undo this.